taggit/utils.py

```diff
@@ -1,7 +1,10 @@
 from __future__ import unicode_literals
 
+from importlib import import_module
+
 from django import VERSION
 from django.utils import six
+from django.conf import settings
```

Comment:
Sure. In a new commit or should I squash all the commits into a single one?

On Mon, Jan 18, 2016 at 4:03 PM, Corey Farwell <notifications@github.com>
wrote:

> In taggit/utils.py
> <https://github.com/alex/django-taggit/pull/232#discussion_r50058630>:
>
> >  from django import VERSION
> >  from django.utils import six
> > +from django.conf import settings
>
> Nit: can you alphabetize this?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/alex/django-taggit/pull/232/files#r50058630>.
>